system/Makefile.am

```diff
@@ -3,6 +3,7 @@
 #
 MAINTAINERCLEANFILES= $(srcdir)/Makefile.in
 CLEANFILES = \
+	netdata.logrotate \
```

Comment:
It is according to 'ls' here...

$ ls system/ | grep in
Makefile.in
netdata.logrotate.in
netdata-openrc.in
netdata.service.in
$

but I guess that's using the locale. <sigh> I'll update it again so it's ascii-sorted instead.....